app/src/main/kotlin/com/wire/android/ui/home/sync/FeatureFlagNotificationViewModel.kt

```diff
@@ -113,6 +118,29 @@ class FeatureFlagNotificationViewModel @Inject constructor(
         }
     }
 
+    private suspend fun setSelfDeletedMessagesFeatureFlag() {
+        viewModelScope.launch {
+            observeSelfDeletingMessages().collect { selfDeletingMessagesStatus ->
```

Comment:
It states to setSelfDeletedMessagesFeatureFlag, but in the end it observes self deleting messages which are emitting selfDeletingMessagesStatus 🤔